modules/statistics/statistics.js

```diff
@@ -33,20 +33,46 @@ let isCallstatsLoaded = false;
  * downloading their API as soon as possible and (2) do the downloading
  * asynchronously.
  *
- * @param customScriptUrl
+ * @param options to use for downloading and initializing callstats backend
```

Comment:
please describe structure of the 'options' argument and that it contains 'customScriptUrl'